restdocs-api-spec-openapi-generator/src/main/kotlin/com/epages/restdocs/apispec/openapi2/OpenApi20Generator.kt

```diff
@@ -412,7 +420,7 @@ object OpenApi20Generator {
         }
     }
 
-    private fun requestParameterDescriptor2Parameter(parameterDescriptor: ParameterDescriptor): QueryParameter {
+    private fun requestParameterDescriptor2QueryParameter(parameterDescriptor: ParameterDescriptor): QueryParameter {
```

Comment:
This could be combined with the function below by using generics, but the current solution is also fine and expressive

Reply:
Great, so I will leave it as is.